src/Simulations/time_step_wizard.jl

```diff
@@ -33,7 +33,9 @@ Base.summary(wizard::TimeStepWizard) = string("TimeStepWizard(",
 Callback for adapting simulation to maintain the advective Courant-Freidrichs-Lewy (CFL)
 number to `cfl`, the `diffusive_cfl`, while also maintaining `max_Δt`, `min_Δt`, and
 satisfying `max_change` and `min_change` criteria so that the simulation's timestep
-`simulation.Δt` is not adapted "too quickly".
+`simulation.Δt` is not adapted "too quickly". In other words, `max_change` is the maximum
+and `min_change` is the minimum relative change of the time step and are, therefore, non-dimensional
+(`min_change * old_Δt ≤ new_Δt ≤ max_change * old_Δt`).
```

Comment:
Note there is a grammatical error in the above as well. If we're seeking improved clarity, I suggest changing to something like

```julia
Callback function that adapts the simulation time step to a target
advective Courant-Freidrichs-Lewy (CFL) number `cfl`, within the limits

    max(max_Δt, max_change * previous_Δt) ≤ new_Δt ≤ min(min_Δt, min_change * previous_Δt)

where `new_Δt` is the new time step calculated by `TimeStepWizard`.
```

Or something along those lines...